components/AddLiquidityModal/index.tsx

```diff
@@ -128,8 +128,6 @@ const AddLiquidityContent = ({ open, onClose, poolData }: Props) => {
     const { isLoggedIn: loggedIn } = useGetLoginInfo();
     const { address, account } = useGetAccountInfo();
     const proxy = getProxyProvider();
-    // const provider = dapp.dapp.provider;
-    const [rates, setRates] = useState<BigNumber[] | undefined>(undefined);
     const { pool, poolStats, liquidityData } = poolData;
 
     // reset when open modal
```

Comment:
@haoquang24 use `ESDTTransfer` to deposit 1 token to pool, instead of `MultiESDTNFTTransfer`.